bin/git-guilt

```diff
@@ -44,10 +44,10 @@ for file in $(git diff --name-only "$@")
 do
     test -n "$DEBUG" && echo "git blame $file"
     # $1 - since $2 - until
-    git blame $NOT_WHITESPACE --line-porcelain "$1" -- "$file" 2> /dev/null |
+    git blame "$NOT_WHITESPACE" --line-porcelain "$1" -- "$file" 2> /dev/null |
```

Comment:
These new changes will cause `git blame` to fail whenever `-w` (or `--ignore-whitespace`) is not supplied at the command line.